examples/http-cache-middleware/README.md

```diff
@@ -0,0 +1,28 @@
+# Http Cache Example
+
+This example demonstrates how to use the http cache middleware in Dumbo 
+
+## Running the Example
+
+1. Install dependencies:
+
+   ```bash
+   composer install
+   ```
+
+2. Start the server:
+
+   ```bash
+   composer start
+   ```
+
+3. Access the page:
+[Click here to access the page](http://localhost:8000/greet/dumbo)
+
+4. Refresh the page to see the cache in action.
```

Comment:
Perhaps we can update this to use curl like the other examples?

```bash
curl -i http://localhost:8000/greet/dumbo
```

Note the ETag in the response headers.

Then, simulate a refresh by including the ETag:

```bash
curl -i -H "If-None-Match: W/\"...\"" http://localhost:8000/greet/dumbo
```